examples/pokemon-service-common/tests/plugins_execution_order.rs

```diff
@@ -64,10 +64,7 @@ struct SentinelPlugin {
 
 impl SentinelPlugin {
     pub fn new(name: &'static str, output: Arc<Mutex<Vec<&'static str>>>) -> Self {
-        Self {
-            name,
-            output: output,
-        }
+        Self { name, output }
```

Comment:
Unrelated clippy lint fix.

Reply:
Isn't clippy run in CI already? https://github.com/awslabs/smithy-rs/blob/c3691087d434a886a259399cea4a9d15b0780a06/examples/Makefile#L23-L24

Reply:
Yeah, I guess I have a newer version locally.